api/requirements.txt

```diff
@@ -0,0 +1,2 @@
+fastapi=0.110.0
+uvicorn=0.29.0
```

Comment:
I missed this. It's supposed to be `==`. It is causing the docker build to fail.

Reply:
I have fixed thisI have fixed this